packages/webos/VoiceControlDecorator/VoiceControlDecorator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {findDOMNode} from 'react-dom';
+import hoc from '@enact/core/hoc';
+
+const VoiceControlDecorator = hoc({}, (config, Wrapped) => {
+	return class extends React.Component {
+		static displayName = 'VoiceControlDecorator'
+
+		static propTypes = {
+			onVoice: PropTypes.func.isRequired
+		}
+
+		componentDidMount () {
+			this.node = findDOMNode(this);
+			if (!(this.node.hasAttribute('data-voice-event-target') || this.node.hasAttribute('data-webos-voice-intent'))) {
```

Comment:
Should this be `data-webos-voice-event-target`?

Reply:
I will change it to data-webos-voice-event-target. (This attribute is not defined by web engine, but it is deeply relate to webOSVoice event. So, it looks better in terms of consistency as you mentioned.) Thanks~